app/src/main/java/se/nullable/flickboard/KeyboardService.kt

```diff
@@ -164,8 +172,13 @@ class KeyboardService : InputMethodService(), LifecycleOwner, SavedStateRegistry
                 FlickBoardParent {
                     ProvideDisplayLimits {
                         var emojiMode by remember { mutableStateOf(false) }
+                        val warningMessageScope = rememberCoroutineScope()
+                        val warningSnackbarHostState = remember { SnackbarHostState() }
                         val appSettings = LocalAppSettings.current
                         val onAction: (Action) -> Unit = { action ->
+                            warningMessageScope.launch {
+                                warningSnackbarHostState.currentSnackbarData?.dismiss()
+                            }
```

Comment:
This may be extraneous since the Snackbar will time out, but my thinking was to get it out of the way if someone resumes typing before it expires.